realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -189,14 +189,22 @@ public boolean isManaged() {
      */
     @Override
     public boolean contains(Object object) {
-        boolean contains = false;
-        if (isLoaded() && object instanceof RealmObjectProxy) {
-            RealmObjectProxy proxy = (RealmObjectProxy) object;
-            if (realm.getPath().equals(proxy.realmGet$proxyState().getRealm$realm().getPath()) && proxy.realmGet$proxyState().getRow$realm() != InvalidRow.INSTANCE) {
-                contains = (table.sourceRowIndex(proxy.realmGet$proxyState().getRow$realm().getIndex()) != TableOrView.NO_MATCH);
```

Comment:
we're not using `Java_io_realm_internal_TableView_nativeFindBySourceNdx` for table views? and instead doing a linear comparison with all `RealmResults` elements? what's the performance impact ?